src/app/Signup/page.module.css

```diff
@@ -0,0 +1,116 @@
+.SignUpPageContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  width: 100vw;
+}
+
+.EmailContainer {
+  align-items: center;
+  margin-bottom: 1rem;
+}
+.PasswordContainer {
+  align-items: center;
+  margin-bottom: 1rem;
+}
+.ConfirmPasswordContainer {
+  align-items: center;
+  margin-bottom: 2rem;
+}
+.TextBoxLabel {
+  margin-bottom: 0.2rem;
+  display: flex;
+}
+.span {
+  display: flex;
+  justify-content: center;
+}
+
+.CreateAccount {
+  margin: 3rem;
+  font-size: 3rem;
+  margin-bottom: 3rem;
+  margin-top: 2rem;
+}
+.lgnbttncontainer,
+.signbttncontainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@media (min-width: 300px) and (max-width: 380px) {
+  .CreateAccount {
+    text-align: center;
+    font-size: 1.8rem;
+    margin-bottom: 1rem;
+    margin-top: 2rem;
+  }
+  .EmailContainer {
+    align-items: center;
+  }
+  .PasswordContainer {
+    align-items: center;
+  }
+  .ConfirmPasswordContainer {
+    align-items: center;
+  }
+  .span {
+    margin-top: 1px;
+  }
+
+  .lgnbttncontainer {
+    margin-bottom: 3rem;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 1024px) {
+  .CreateAccount {
+    text-align: center;
+    font-size: 3rem;
+    margin-bottom: 10px;
+    margin-top: 2rem;
+  }
+  .EmailContainer {
+    margin-bottom: 2rem;
+    margin-top: 4rem;
+  }
+  .PasswordContainer {
+    margin: 2rem 0;
+  }
+  .ConfirmPasswordContainer {
+    margin: 2rem 0;
+  }
+  .signbttncontainer {
+    margin-top: 4rem;
+  }
+}
+
+@media (min-width: 390px) and (max-width: 430px) {
+  .CreateAccount {
+    text-align: center;
+    font-size: 2.5rem;
+    margin-bottom: 10px;
+    margin-top: 2rem;
+  }
+  .EmailContainer {
+    margin: 2rem 0;
+  }
+  .PasswordContainer {
+    margin: 2rem 0;
+  }
+  .ConfirmPasswordContainer {
+    margin: 2rem 0;
+  }
+  .signbttncontainer {
+    margin-top: 4rem;
+  }
+}
+
+/* for surface duo only, yeah i guess its special like that :/ */
```

Comment:
nice comment LMFAO, btw when u implement the text align change above u can delete this media query